rotkehlchen/fval.py

```diff
@@ -137,6 +137,10 @@ def __rmod__(self, other: AcceptableFValOtherInput) -> 'FVal':
         evaluated_other = _evaluate_input(other)
         return FVal(self.num.__rmod__(evaluated_other))
 
+    def __round__(self, ndigits: int) -> 'FVal':
```

Comment:
docstring. why the assertion? negative values work fine for round. Are you sure we got nothing else for this? We must have done somethingsimilar elsewhere.

Reply:
I've been checking other places where we round and we don't seem to have this

Reply:
you did not add a docstring